requirements-transitive.txt

```diff
@@ -17,14 +17,14 @@ certifi==2020.12.5
 cffi==1.14.5
```

Comment:
Is there a need for these upgrades to happen as part of this change besides the m1 stuff (which shouldn't be needed when using docker compose to do the testing)? generally I'd opt to separate out upgrades from functional changes in case there's a hidden breaking change to make it easier to roll back.

Reply:
True, I can move to a separate PR.